hydroflow_plus/src/snapshots/hydroflow_plus__profiler__tests__predicate_pushdown_through_map.snap.new

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
you prob need to regenerate snapshots: `cargo insta test -p hydroflow_plus` and then `cargo insta review`

you may need to `cargo install cargo-insta --locked` if the command is not found